packaging/hudi-spark-bundle/pom.xml

```diff
@@ -294,6 +295,10 @@
                   <shadedPattern>org.apache.hudi.org.apache.hadoop.metrics2.util.MetricSampleQuantiles
                   </shadedPattern>
                 </relocation>
+                <relocation>
```

Comment:
If it is being shaded in spark bundle, then please ensure Avro is relocated in every other bundle, e.g. Hudi-hive-sync-bundle otherwise the two bundles (one with shaded Avro and another with unshaded Avro) will run into conflict.
@nsivabalan Do you see any issues with relocating Avro in hudi-hive-sync-bundle?